cmd/internal/datastore/os/transaction_test.go

```diff
@@ -0,0 +1,84 @@
+package os
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/mailchain/mailchain/cmd/internal/datastore"
+	"github.com/mailchain/mailchain/encoding/encodingtest"
+	"github.com/mailchain/mailchain/internal/address/addresstest"
+	"github.com/mailchain/mailchain/internal/protocols"
+	"github.com/mailchain/mailchain/internal/protocols/ethereum"
+	"github.com/spf13/afero"
+)
+
```

Comment:
Try adding this function to check if the file system / `afero` has any files in it. 

```
func readDir(fs afero.Fs, root string) []string {
	var files []string
	f, err := fs.Open(root)
	if err != nil {
		panic(err)
	}
	fileInfo, err := f.Readdir(-1)
	f.Close()
	if err != nil {
		panic(err)
	}
	for _, file := range fileInfo {
		files = append(files, file.Name())
	}
	return files
}
```

Reply:
Have moved this function to a separate test package